composer.json

```diff
@@ -50,6 +50,7 @@
         "liip/imagine-bundle": "1.3.0",
         "imagine/imagine": "0.6.2",
         "monolog/monolog": "1.22.0",
+        "ocramius/proxy-manager": "2.0.4",
```

Comment:
this is required by doctrine/migrations 1.5.0
I locked the version as this is what we do currently

(personally I don't like to do that, so I'd be happy if you want me to drop it)